daemon/logger/copier_test.go

```diff
@@ -102,6 +106,50 @@ func TestCopier(t *testing.T) {
 	}
 }
 
+// TestCopierTimestamp tests that Copier is using a time function to add timestamp to a message
+func TestCopierTimestamp(t *testing.T) {
+	timeFunc := func() time.Time {
+		return time.Date(2017, time.June, 0, 0, 0, 0, 0, time.UTC)
+	}
+	var stdout bytes.Buffer
+	if _, err := stdout.WriteString("hello world\n"); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I don't think `bytes.Buffer.WriteString()` can actually return an error, it always returns `nil`.

You can also just use the string constructor to initialize it: https://golang.org/pkg/bytes/#NewBufferString

Instead of splitting it across two lines